Wireframe/index.html

```diff
@@ -1,33 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
```

Comment:
Why did you change the indentation of a bunch of this file?

In general seeing what's nested inside what makes it a lot easier to read.